gcovr/configuration.py

```diff
@@ -421,6 +421,16 @@ def choose(_cls, choices, default=None):
              "The --root is the default --filter.",
         default='.',
     ),
+    GcovrConfigOption(
+        "add_tracefile", ["-a", "--add-tracefile"],
+        help="Combine the coverage data from JSON files. "
```

Comment:
> Perhaps the main() should have additional checks so that it can warn when unused options like filters were specified on the command line (but not when they are just loaded from the config file).

OK, just mention about that no coverage data will be collected from gcov as it is already done for other options like - - branches, - - sort-uncovered, - -sort-percentage or do something more sophisticated in the main()? What with other options like for example calling the gcovr to XML output with unused the - - sort-uncovered option?

Reply:
Hmm, let's forget about adding checks, since the other options don't have them either. That's a problem for a future enhancement.

So as long as the --add-tracefile help message is updated to clearly note that it will not run gcov to collect new coverage data, that is fine by me.